### PR TITLE
Add pre-baked Centos Image

### PR DESCRIPTION
**Pull request by kashifest**
This PR adds prebaked Centos node image URL. 
The preBaking is done [here](https://github.com/Nordix/airship-dev-tools/pull/231)

### CODE REVIEW

**Review by Jaakko-Os — COMMENTED** [review]
lgtm

File: vm-setup/roles/v1aX_integration_test/vars/main.yml
Comment (fmuyassarov) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:64]: Should this also be changed to the similar name as the default value of IMAGE_NAME_CENTOS?

Comment (fmuyassarov) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:64]: Should this also be changed to the similar name`-raw.img `as the default value of IMAGE_NAME_CENTOS?

**Review by fmuyassarov — COMMENTED** [review]
Please make sure to update the vars.md with corresponding changes.Otherwise lgtm.

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc
Comment (maelk) [inline vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc:51]: you need to do something similar for the workers

Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc:51]: Ow, good catch, it might improve the timing more

**Review by namnx228 — APPROVED** [review]

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc
Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc:51]: done

File: vm-setup/roles/v1aX_integration_test/vars/main.yml
Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/vars/main.yml]: Done

**Review by maelk — APPROVED** [review]

**Comment by kashifest** [comment]
@fmuyassarov Document update done, are those ok or do I need to add something somewhere else also?

**Review by furkatgofurov7 — COMMENTED** [review]
lgtm
Since others have comments, I will leave it to them to lgtm.

**Pull request merged**
